src/components/entity/list.vue

```diff
@@ -458,6 +498,105 @@ export default {
       // less than the lowest size option, hence we don't need to make a request.
       if (this.odataEntities.count < this.pageSizeOptions[0]) return;
       this.fetchChunk(false);
+    },
+    clearSelectedEntities() {
+      this.odataEntities.value.forEach(e => { e.__system.selected = false; });
+      this.selectedEntities.clear();
+    },
+    requestBulkDelete() {
+      const uuids = Array.from(this.selectedEntities);
+      // TODO: disable the whole table
+
+      const bulkDelete = () => this.request({
+        method: 'POST',
+        url: apiPaths.entities(this.projectId, this.datasetName, '/bulk-delete'),
+        alert: false,
+        data: {
+          ids: uuids
+        }
+      });
+
+      const onSuccess = () => {
+        if (this.deletedEntityCount.dataExists) this.deletedEntityCount.value += uuids.length;
+
+        uuids.forEach(uuid => this.odataEntities.removedEntities.add(uuid));
+
+        this.deletedEntities = [];
+        this.deletedEntities.push(...this.odataEntities.value.filter(e => uuids.includes(e.__id)));
```

Comment:
Instead of:

```js
this.deletedEntities = [];
this.deletedEntities.push(...anotherArray);
```

I think we can just do:

```js
this.deletedEntities = anotherArray;
```

Reply:
it makes me nervous to assign one array to another because if I modify `anotherArray` then `this.deletedEntities` will be modified.